app.js

```diff
@@ -75,7 +75,9 @@ app.use(function (req, res, next) {
     return jsonParser(req, res, next);
 });
 
-app.use(passportAuth.initialize());
+if (process.env.NODE_ENV !== "test") {
+  app.use(passportAuth.initialize());
```

Comment:
Instead of conditional here in code proper I would mock lib/controllers/helpers/passport.js in the tests. Same with changes in lib/controllers/helpers/auth.js.